pkg/containerd/events.go

```diff
@@ -192,9 +188,17 @@ func (e *EventMonitor) IsServing(ctx context.Context) error {
 
 // Close closes the client connection to the API server.
 func (e *EventMonitor) Close() error {
-	e.portTracker.RemoveAll()
+	err := e.containerdClient.Close()
+	if err != nil {
+		return fmt.Errorf("failed to close containerd client: %w", err)
```

Comment:
If we return here, we never remove the ports. Is that the correct behaviour?

Reply:
Yes, because the process will terminate after this.

Reply:
But the ports (in `netsh portproxy`) are still there? We can't terminate that.